server/offenceCodeDecisions/Decision.ts

```diff
@@ -14,7 +14,7 @@ export default class Decision {
 
   private decisionTitle: Title
 
-  private decisionPage: Page
+  private decisionType: DecisionType = DecisionType.DEFAULT
```

Comment:
We have a decision type that we can switch on for when we need more than just a radio button